docs/docs/guides/remote-machine-learning.md

```diff
@@ -37,8 +39,26 @@ volumes:
   model-cache:
 ```
 
-Please note that version mismatches between both hosts may cause instabilities and bugs, so make sure to always perform updates together.
+3. Start the remote machine learning container by running `docker compose up -d`
+
+:::info
+Version mismatches between both hosts may cause bugs and instability, so remember to update this container as well when updating the local Immich instance.
+:::
+
+4. Navigate to the [Machine Learning Settings](https://my.immich.app/admin/system-settings?isOpen=machine-learning)
+5. Click _Add URL_
+6. Fill the new field with the URL to the remote machine learning container. In the simplest case where the server is on the same local network, this may use the server's private IP. For example, if the private IP of the remote server is 192.168.0.50, the resulting URL will be `http://192.168.0.50:3003`. More advanced configuration is left as an exercise to the reader
```

Comment:
I think this could be confusing since you cannot actually use your local ip from a docker container but instead need to use the local ip in the docker network

Reply:
Hmm, good point. Do you have any suggestions to guide the user here?

Reply:
I think we need, at a minimum, a format example. Something like `http://ip:port`. It's not always obvious what format systems expect hostname/URL to be passed in as - see, our earlier server URL that didn't enforce HTTP.